docs/features/multidomain.rst

```diff
@@ -0,0 +1,333 @@
+                egress = 200, -- kbit/s
+                ingress = 3000, -- kbit/s
+            },
+        },
+        
+        autoupdater = {
+            branch = 'stable',
+            
+            branches = {
+                stable = {
+                    name = 'stable',
+                    mirrors = {'http://update.example.org/stable/sysupgrade'},
+                    good_signatures = 2,
+                    pubkeys = {
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Alice
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Bob
+                        'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', -- Mary
+                    },
+                },
+            },
+        },
+    }
+
+domains/alpha_centauri.conf
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: lua
+
+    {
+        -- multiple codes/names can be defined, the first one is the primary name
+        -- additional aliases can be defined
+        domain_names = {
+            alpha_centauri = 'Alpha Centauri',
+            rigil_kentaurus = 'Rigil Kentaurus',
+            proxima_centauri = 'Proxima Centauri',
+        },
+        
+        -- 32 byte random data in hexadecimal encoding
+        -- Can be generated using: echo $(hexdump -v -n 32 -e '1/1 "%02x"' </dev/urandom)
+        domain_seed = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+        
+        -- unique network prefixes per domain
+        prefix4 = '10.xxx.0.0/20',
+        prefix6 = 'fdxx:xxxx:xxxx:xxxx::/64',
+        
+        next_node = {
+            ip4 = '10.xxx.yyy.zzz',
+            ip6 = 'fdxx:xxxx:xxxx:xxxx::xxxx',
+        },
+        
+        wifi24= {
+            ap = {
+                ssid = "alpha-centauri.example.org",
+                channel = 5,
+            },
+            mesh = {
+                id = 'mesh-alpha-centauri',
+            },
+        },
+
+        wifi5= {
+            ap = {
+                ssid = "alpha-centauri.example.org",
+                channel = 36,
+            },
+            mesh = {
+                id = 'mesh-alpha-centauri',
+            },
+        },
+        
+        mesh_vpn = {
+            fastd = {
+                groups = {
+                    backbone = {
+                        peers = {
+                            peer1 = {
+                                key = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+                                remotes = {'"peer1.example.org" port xxxxx'},
+                            },
+                            peer2 = {
+                                key = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+                                remotes = {'"peer2.example.org" port xxxxx'},
+                            },
+                        },
+                    },
+                },
+            },
+        },
+    }
```

Comment:
I think it would make sense to provide these examples as plain config files and just include them from the RST, like it is done with the single-domain example site.conf/site.mk.